readers-frontend/src/components/HeaderNavbar.astro

```diff
@@ -0,0 +1,124 @@
+---
+import type { categories } from "@prisma/client";
+import prisma from "../prisma";
+import { Icon } from "astro-icon";
+
+interface Props {
+  active: string;
+}
+
+let menuCategories: categories[] = await prisma.categories.findMany({
+  where: {
+    categories_showMenu: true,
+    categories_nestUnder: null,
+    NOT: { categories_name: "muse" }, // Added manually because of image layout
```

Comment:
I thought we agreed the ticket to be nouse desktop only. Also, not including muse here is because the muse element uses an image rather than text and we'd need a conditional branch on every iteration to inject the image

Reply:
Inside the `a` tag we just display the `menuItem.categories_displayName`. For muse we use an `astro-icon::Icon`. If we need to check for muse we need to check within `menuCategories.map(...)`. That's a check for each iteration. But yeah I can just put it there, no big deal